tests/unit/reset.js

```diff
@@ -22,3 +22,20 @@ test('reset abort request', function (t) {
 
   t.is(state.request.abort.callCount, 1, 'request aborted')
 })
+
+test('reset interval to default', function (t) {
+  var intervalValue = 30000
+  var state = {
+    method: 'HEAD',
+    url: 'https://example.com/ping',
+    cache: {},
+    interval: intervalValue
```

Comment:
state internally should never be `state.interval: 30000`, it should always be split up between `state.interval.connected` and `state.interval.disconnected`. 

But `options.interval`, which can be passed to both the `new ConnectioStatus(options)` constructor and the the `connectionStatus.reset(options)` method, can be a number and will then be split up into `connected` / `disconnected` before setting the internal state. Makes sense?